architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management for Kubernetes, or as a standalone operator that is supported as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription.
+
+If you use Red Hat Advanced Cluster Management, multicluster engine is included and provides the following capabilities:
+
+* klusterlet agent, which registers managed clusters to the hub.
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of OpenShift, such as SNO on bare metal. The Infrastructure Operator includes Central Infrastructure Management (CIM), which provides an interactive user interface in the Red Hat Advanced Cluster Management console where you can provision clusters.
+* Open cluster management, which provides resources to manage multiple Kubernetes clusters.
+
+If you use {product-title} 4.8.2 or later, you can separately install multicluster engine to access the following capabilities:
+
+* Hive, which provisions and manages self-managed OpenShift clusters to the hub.
+* Zero touch provisioning (ZTP), which provides full automation for cluster creation on bare metal and vSphere with minimal intervention. ZTP also offers a GitOps workflow for managing deployments.
+* Support for hosted control planes for {product-title}.
```

Comment:
would it be OK to include:
"(from the **HyperShift** project) somewhere on this page so that if people search for that term in the doc, it will land them here?"

Reply:
Seems odd that we give a 1-liner on what Hosted control planes does (whereas the other bullets do).  Suggesting the following instead:

Hosted control planes provides a centralized control plane for OpenShift clusters which enables you to operate multiple clusters in a hyperscale manner. Hosted control planes is available on {product-title}.